### PR TITLE
Feature 487 ii

### PR DESCRIPTION
**Pull request by gato**
Hi Max,
I've added detailed timing information to FlowDetailView, as you requested in #487 
They are in the same order that where in the screenshot and then are sorted by timestamp like is done in https://github.com/mitmproxy/mitmproxy/blob/master/libmproxy/web/static/app.js#L1084 
there is no delta for now, but can [easily] be added later.
Regards.
Marcelo

### CODE REVIEW

File: libmproxy/utils.py [hunk 1]
```diff
@@ -15,7 +15,10 @@ def format_timestamp(s):
     d = datetime.datetime.fromtimestamp(time.mktime(s))
     return d.strftime("%Y-%m-%d %H:%M:%S")
 
-
+def format_timestamp_with_milli(s):
+    d = datetime.datetime.fromtimestamp(s)
+    return d.strftime("%Y-%m-%d %H:%M:%S.")+str(d.microsecond/1000).zfill(3)
```
Comment (mhils) [inline libmproxy/utils.py]: Simplify to `"%Y-%m-%d %H:%M:%S.%f"[:-3]` ?

File: libmproxy/console/flowdetailview.py [hunk 2]
```diff
@@ -35,17 +35,20 @@ def flowtext(self):
         title = urwid.AttrWrap(title, "heading")
         text.append(title)
 
-        if self.flow.server_conn:
+        cc = self.flow.client_conn
+        sc = self.flow.server_conn
+        req = self.flow.request;
```
Comment (mhils) [inline libmproxy/console/flowdetailview.py]: Remove `;` :smiley:

**Comment by mhils** [comment]
Thanks, great PR as always. I have two minor complaints which usually wouldn't stop me from merging it directly, but we're currently lacking simple tasks for potential novice GSoC students. Maybe someone wants to pick this PR up, make the minor adjustments mentioned above and then send another PR? :smiley:

This is mostly about figuring out how to add another git remote, do the merge, add your changes and push a PR yourself.

Cheers,
Max

**Comment by mhils** [comment]
@gato: To clarify: If noone picks that up I'll quickly do the adjustments. No further action required from your side, great work. :smile:

File: libmproxy/console/flowdetailview.py (hunk 2 above)
Comment (gato) [inline libmproxy/console/flowdetailview.py]: so many languages in my coding life, made my fingers have code completion, but some times they mix things up ;)

**Pull request merged**